custom_dc/README.md

```diff
@@ -0,0 +1,262 @@
+# Custom Data Commons
+
+A custom Data Commons can hold new data and include UI modifications that are
+custom to a third party's need.
+
+This document illustrates development and deployment details for a custom Data
+Commons instance.
+
+## Prerequisites
```

Comment:
It might be nice to have a visual of the 3-4 approaches (local, cloud with SQLlite on cloud disk, cloud with google sql) at the top.